apps/site/pages/en/learn/userland-migrations/introduction.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Userland Migrations
+layout: learn
+authors: JakobJingleheimer, AugustinMauroy
+---
+
+![Node.js Userland Migrations](https://raw.githubusercontent.com/nodejs/userland-migrations/main/.github/assets/Userland-Migration-Tagline.png)
+
+# Userland Migrations
+
+Node.js provides migrations for "userland" (what you write vs node's own) source-code to facilitate adoption of new features and upgrading source-code affected by breaking changes. These are done in collaboration with [`codemod`](https://docs.codemod.com/introduction), who also work with other major projects like Next.js, React, and Tailwind. Node.js's migrations live in the [`nodejs/userland-migrations`](https://github.com/nodejs/userland-migrations) repository and are overseen by the `@nodejs/userland-migrations` team.
+
+<!--
+maintainer node: https://codemod.link/nodejs-official is pointing to legacy codemod registry
+it's will point to new new one once codemod change the redirection of the short link
+-->
+
+Official migrations are published under the `@nodejs` namespace within the [codemod registry](https://codemod.link/nodejs-official). These have been reviewed and/or authored by Node.js members. There are also unofficial migrations available which have not been reviewed by Node.js.
+
+## Our goal
+
+First our goal is to help you to migrate your codebase to the latest Node.js version. We want to make it easier for you to adopt new features, deprecations, and breaking changes. In summary, we want to help you to migrate your codebase to the latest Node.js version.
+
+Second, we want to help you to adopt native Node.js features that are already available in the ecosystem. For example, if you are using a library that provides a feature that is already available in Node.js, we want to help you to migrate to the native Node.js feature.
+
+## How to use a Codemod
+
+To use a codemod, you can run the following command in your terminal:
+
+```bash displayName="npm"
+npx codemod@next <codemod-name>
+```
```

Comment:
Let's only showcase npm

Reply:
Why ? On download we explain how to install yarn pnpm so why not on this page show them